projects/laji-ui/src/lib/directives/a11y-click.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, ElementRef, HostListener } from '@angular/core';
+
+enum KEY_CODE {
+  ENTER = 13
+}
+
+@Directive({
+  selector: '[luA11yClick]'
+})
+export class a11yClickDirective {
+  constructor(private el: ElementRef) {}
+
+  @HostListener('window:keyup', ['$event'])
+  keyEvent(event: KeyboardEvent) {
+    if (event.target === this.el.nativeElement && event.keyCode === KEY_CODE.ENTER) {
```

Comment:
There was a small error here that caused the performance issues: the event handler was bound to "window" rather than the host element. The issue is now fixed :)

Reply:
ok that helps with the callbacks yes but this would still register 176 individual event listeners on the observation page. I know on older browsers that causes performance issues, but since we dropped IE11, this is okay.